src/Components/Numbers.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import CSS from "csstype";
+import "../App.css";
+
+interface Props {
+  handleClick(el: string): void;
+}
+
+const Numbers = ({ handleClick }: Props) => {
+  const numbers: string[] = [
```

Comment:
I think I'd define this array outside the component. I'm not sure if React or the Browser will optimise it away under the hood, but if you stick a breakpoint on this line it fires every time a button is clicked.